chromium_src/BUILD.gn

```diff
@@ -414,4 +413,8 @@ source_set("chrome_spellchecker") {
     "//components/spellcheck/common",
     "//components/spellcheck/renderer",
   ]
+
+  if (enable_builtin_spellchecker) {
+    public_deps += [ "//chrome/common:constants" ]
```

Comment:
can you declare `public_deps = []`


and move this inside the previous `enable_builtin_spellchecker` block, would like to avoid repeated condition blocks.

Reply:
Fixed up.